### PR TITLE
Small fixes to 2.1 and 4.1

### PR DESCRIPTION
**Pull request by canyon289**
@ColCarroll 
Could you verify that the indicies autocorrelation statement is correct when you get a chance? The `chain[0:(500-40)]` and `chain[40:500]`

> If we look at value 40, the value indicates `chain[0:(500-40)]` still has about a .75 value of correlation with `chain[40:500]`.

image

### CODE REVIEW

**Comment by canyon289** [comment]
Fixes https://github.com/arviz-devs/bayesian-model-evaluation/issues/43

**Comment by canyon289** [comment]
Fixes https://github.com/arviz-devs/bayesian-model-evaluation/issues/19

**Comment by canyon289** [comment]
Fixes https://github.com/arviz-devs/bayesian-model-evaluation/issues/18

File: notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb [hunk 1]
```diff
@@ -581,7 +581,7 @@
    "source": [
```
Comment (ColCarroll) [inline notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb:581]: <p>the sum should start from `i=0`, and the elements on the bottom should be squared.</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (ColCarroll) [inline notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb:581]: <p>I think this is a really good example, and perhaps even one we should work through with a pen and paper! It isn't _so_ straightforward to read this chart -- try</p><p><br></p><p>```</p><p>chain = np.arange(500)</p><p>az.plot_autocorr(chain, max_lag=500);</p><p>```</p><p><br></p><p>what interpretation do we have that `n=182` has autocorrelation 0?</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (ColCarroll) [reply notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb:581]: (to be clear, i don't have a good answer for this right now)

Comment (canyon289) [reply notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb:581]: Got more review than I was bargaining for :D Will fix tonight. Thank you

**Comment by canyon289** [comment]
Added issue https://github.com/arviz-devs/bayesian-model-evaluation/issues/46

Merging so we can keep working without merge conflicts

**Pull request merged**